tests/api/test_committee_meeting_attendance_excel_export.py

```diff
@@ -0,0 +1,21 @@
+from tests import PMGTestCase
+from tests.fixtures import dbfixture, CommitteeMeetingData
+
+
+class TestCommitteeMeetingAttendanceExcelExport(PMGTestCase):
+    def setUp(self):
+        super(TestCommitteeMeetingAttendanceExcelExport, self).setUp()
+
+        self.fx = dbfixture.data(CommitteeMeetingData)
+        self.fx.setup()
+
+    def tearDown(self):
+        self.fx.teardown()
+        super(TestCommitteeMeetingAttendanceExcelExport, self).tearDown()
+
+    def test_get_committee_meeting_attendance_xlsx_smoke_test(self):
+        res = self.client.get(
+            "committee-meeting-attendance/data.xlsx",
+            base_url="http://api.pmg.test:5000/",
+        )
+        self.assertEqual(200, res.status_code)
```

Comment:
perhaps an easy improvement to this test is to just open it with openpyx or whatever the library is already in this project to see if it fails. The simplest operation that fails if the response is _not_ an excel file.